src/NServiceBus.Core/Performance/Statistics/CriticalTime/CriticalTimeMonitoringConfig.cs

```diff
@@ -5,6 +5,10 @@ namespace NServiceBus
     /// <summary>
     /// Provide configuration options for monitoring related settings.
     /// </summary>
+    [ObsoleteEx(
+        Message = "Performance counters have been released as a separate package: NServiceBus.WindowsPerformanceCounters",
+        RemoveInVersion = "8",
+        TreatAsErrorFromVersion = "7")]
     public static class CriticalTimeMonitoringConfig
```

Comment:
you should also obsolete the extension method I think as you'll probably never reference this class directly. Have you tested this?